docs/.vuepress/local-components/api/ModuleCard.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="docs-card rounded">
+    <router-link :to="link" class="block p-6">
```

Comment:
```suggestion
    <router-link :to="link" class="block p-4">
```
Along with making the icon float right, this helps make the cards more compact.

Here's a visual diff to show:

<img width="702" alt="Screen Shot 2020-10-21 at 3 32 15 PM" src="https://user-images.githubusercontent.com/3252/96773613-cbd87880-13b2-11eb-9560-14dd798e2c88.png">


<img width="693" alt="Screen Shot 2020-10-21 at 3 33 12 PM" src="https://user-images.githubusercontent.com/3252/96773629-d1ce5980-13b2-11eb-8eab-da5a42672856.png">